### PR TITLE
Fill a form

### PR DESCRIPTION
**Pull request by mafeZapata**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
+    "pretest": "rm -rf cypress/reports/*",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress run",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports"
```
Comment (holgiosalos) [inline package.json]: are these changes required in this step?

File: cypress/page/personal-form.page.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            cy.get(this.hobbiesCheckbox).filter(`:contains("${hobbie}")`).click();
+        });
+        cy.get(this.currentAddressInput).type(personalInformation.currentAddress);
+        cy.get(this.submitBtn).click({force: true});
+    }
+
+    private fillDateInput(date: string){
+        const newdate = new Date(date);
+        cy.get(this.dateOfBirthPicker).click();
+        cy.get(this.datePickerValues[0]).select(`${newdate.getFullYear()}`);
```
Comment (holgiosalos) [inline cypress/page/personal-form.page.ts]: could you convert `datePickerValues` into a dictionary or into an object, so instead of accessing indexes we can access to properties that are more readable, so for example, to access the year, we don use index 0, we use: `this.datePickerValues.year`

File: package.json (hunk 1 above)
Comment (mafeZapata) [reply package.json]: this changes are for the  step 13

Comment (holgiosalos) [reply package.json]: Oh I see so you are including step 13 and 14 in this PR. that's ok but, then you need to add t he posttest script and attach an screenshot of both:

- HTML report
- console report

**Comment by mafeZapata** [comment]
Capture fill a form console
Capture fill a form html

File: package.json [hunk 3]
```diff
@@ -5,30 +5,13 @@
...
-    "practice",
-    "cypress"
-  ],
-  "author": "Maria Fernanda Zapata mafez0427@gmail.com mafeZapata",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/mafeZapata/cypress-training/issues"
-  },
-  "homepage": "https://github.com/mafeZapata/cypress-training#readme",
-  "devDependencies": {
```
Comment (holgiosalos) [inline package.json]: why you delete this?

**Review by holgiosalos — COMMENTED** [review]
Why you decide don't suffer with the date 😢 
I accept the validation of the modal as a tribute 😄

File: package.json (hunk 3 above)
Comment (mafeZapata) [reply package.json]: I have a problem with the CI with the lint but i upload the file and resolve this problem

File: package.json [hunk 4]
```diff
@@ -8,6 +8,7 @@
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
     "test": "cypress run"
+
```
Comment (holgiosalos) [inline package.json]: here you are missing the scripts related to clean, generate, combine the reports as well as the pre and post test scripts

**Review by manuelq12 — APPROVED** [review]
